### DISCUSSION TITLE
Disable console log "x store installed" messages

### DISCUSSION THREAD

**Discussion by martinszeltins** [vuejs/pinia#2703] [Help and Questions] [answered]
### What problem is this solving

I can't find anywhere how to disable these console.log messages:

````
🍍 "session" store installed 🆕
🍍 "app" store installed 🆕
🍍 "printer" store installed 🆕
````

It would be nice to be able to turn these notifications off about installed stores. They are just spamming my console.

### Proposed solution

Perhaps a config options to turn them off?

**Comment by posva** [comment]
IIRC there is an option in the devtools options for it. If not, i will add this to the list of the devtools issue

**Reply by posva** [reply, marked as answer]
It's here:

<img width="326" alt="Screenshot 2024-06-26 at 07 24 26" src="https://github.com/vuejs/pinia/assets/664177/762de1cc-d377-4cd6-865e-b9e2bc6dff62">

<img width="853" alt="Screenshot 2024-06-26 at 07 24 39" src="https://github.com/vuejs/pinia/assets/664177/cd906054-e935-4880-87a0-f497eaaef5dc">

**Reply by martinszeltins** [reply]
@posva but this configuration is per-user and not permanent, right? If I clear the cookies/localStorage - the settings will be gone. It would be nice to configure this in my code and commit to repo.

**Reply by BenJackGill** [reply]
Also toggling that setting has no effect on the messages for me. I am still seeing them.

**Reply by xjsdev** [reply]
if you want the production to be silent, you can try replacing the logging function with a stub:

`// @ts-ignore`
`window.__VUE_DEVTOOLS_TOAST__ = () => {}`

**Reply by weo3dev** [reply]
@xjsdev - can you enlighten me (us) as to where we would put this stub? Much appreciated.

**Reply by xjsdev** [reply]
> @xjsdev - can you enlighten me (us) as to where we would put this stub? Much appreciated.

Put it to code, that loads before client-side logs start(before this module loads) and has access to window global obj.

**Comment by travisricks** [comment]
@posva 
Is it possible to reopen this discussion? I've toggled off that setting but still get the messages in the console.
- Chrome: Version 127.0.6533.89 (Official Build) (arm64)
- Vue.js devtools: 6.6.3

**Reply by parreirat** [reply]
I also can't find the option for toggling this at all, nothing no Pinia section:

![image](https://github.com/user-attachments/assets/3f156d97-0c2c-4338-966d-eb8d9c448a52)

And nothing on the Vue settings side either:

![image](https://github.com/user-attachments/assets/442842fa-e032-48fe-bdab-ea15f4f62247)

What am I missing? Thank you :)

- Version 129.0.6668.60 (Official Build) (arm64)
- Vue DevTools: v7.2.1

**Reply by f1n4l34f** [reply]
Hi, I found it.

![image](https://github.com/user-attachments/assets/f6183866-740e-4951-80d9-ed90b7a62233)

**Reply by martinszeltins** [reply]
@f1n4l34f that setting is not persisted. clear browser cache and you lose it

**Comment by amirafa** [comment]
for newest dev-tool (v 7.6.5)
![1](https://github.com/user-attachments/assets/776dc065-c7d8-41f8-8809-78888e42ba80)
![2](https://github.com/user-attachments/assets/f2fa459f-a2b3-4922-a457-08003c84624a)
![3](https://github.com/user-attachments/assets/38f36bf2-ca8d-4389-b8a6-7b11ddb5db24)
